modules/importer/src/server/mod.rs

```diff
@@ -92,6 +96,9 @@ impl Server {
             runs.retain(|heart| heart.is_beating());
             let count = runs.len();
 
+            // Update metrics
+            running_importers.record(count as _, &[]);
```

Comment:
When there are multiple replicas, won't the idle ones be updating this value to 0 every second?

Reply:
Each instance has its own instance ID.